### PR TITLE
Add refine

### LINKED ISSUE #8831: Refine Icon
### Brand Name

Refine

### Website

https://refine.dev/

### Popularity Metric

Github stars: **11.5k**
Similarweb global rank: **142,586**
Similarweb cat rank: **4,738**

### Official Resources for Icon and Color

logos:

1. Logo -  https://avatars.githubusercontent.com/u/104967037?s=200&v=4
    color- #1890ff

2. Logo - https://refine.dev/img/refine_favicon.png

### Additional Comments

From the looks of it, they have 2 icons right now. One is a full form one that I am unable to have access to as its an imbedded svg.
Regarding colours their official one seems to be #1890ff since their examples in the docs use the first logo that is in the first link that I provided which is off github.

### Contributing

No

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
refinedotdev

**Issue:** closes #8831

**Similarweb rank:**
The similarweb rank is 3,440. See https://www.similarweb.com/website/refine.dev

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website https://refine.dev/about/
Color: `#0B82F0` Taken from https://github.com/refinedev/refine/blob/next/logo.png

### CODE REVIEW

**Comment by LitoMore** [comment]
Hi @omeraplak, could you help to review this icon? Does the icon meet your expectations?

**Comment by omeraplak** [comment]
> Hi @omeraplak, could you help to review this icon? Does the icon meet your expectations? 

Hey,
Thank you for contacting us! I think @recepkutuk can help us!

**Comment by PratyushJayachandran** [comment]
> Hey, Thank you for contacting us! I think @recepkutuk can help us!

Hello @recepkutuk, looking forward to your review before we merge this one.

**Comment by recepkutuk** [comment]
Hi @PratyushJayachandran this looks great, but we dropped the blue color on the logo. Here's the link to our latest brand assets: https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&t=DkDibSQq9GrkqBRE-6

**Comment by PratyushJayachandran** [comment]
> Hi @PratyushJayachandran this looks great, but we dropped the blue color on the logo. Here's the link to our latest brand assets: https://www.figma.com/file/y8lhTyumAN0zYI5nKKLoLN/refine.dev-brand-assets?type=design&t=DkDibSQq9GrkqBRE-6

Thank you so much @recepkutuk. I have updated the PR with new source and guidelines and the color changed too
refinedotdev

**Comment by recepkutuk** [comment]
Thank you @PratyushJayachandran , it's perfect now 👍

File: _data/simple-icons.json
Comment (adamrusted) [inline _data/simple-icons.json:11109]: File no longer exists. We could do with a permalink that the brand redirects to the relevant Figma file.

Comment (adamrusted) [reply _data/simple-icons.json:11109]: IMO I think we're best to link to the homepage, as right-clicking on the logo will send you to the Figma file.

Comment (adamrusted) [reply _data/simple-icons.json:11109]: @recepkutuk - do you have a permalink which redirects to the latest (like refine.dev/brand) that we can use?
It'd make reviewing in the future a _lot_ easier!

Comment (recepkutuk) [reply _data/simple-icons.json:11109]: Sure, here's the link: https://s.refine.dev/refine-brand-assets

Comment (adamrusted) [inline _data/simple-icons.json:11109]:
```suggestion
            "source": "https://s.refine.dev/refine-brand-assets",
            "guidelines": "https://s.refine.dev/refine-brand-assets"
```

File: _data/simple-icons.json
Comment (PeterShaggyNoble) [inline _data/simple-icons.json:11353]:
```suggestion
            "title": "Refine",
```

File: icons/refinedotdev.svg
Comment (PeterShaggyNoble) [inline icons/refinedotdev.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Refine</title><path d="M13.789.422a4.001 4.001 0 0 0-3.578 0l-8 4A4.0011 4.0011 0 0 0 0 8v8c0 1.515.856 2.9 2.211 3.578l8 4a4.001 4.001 0 0 0 3.578 0l8-4A4.0011 4.0011 0 0 0 24 16V8c0-1.515-.856-2.9-2.211-3.578l-8-4ZM8 8c0-2.209 1.791-4 4-4s4 1.791 4 4v8c0 2.209-1.791 4-4 4s-4-1.791-4-4V8Zm6 0c0 1.105-.895 2-2 2s-2-.895-2-2 .895-2 2-2 2 .895 2 2Z"/></svg>
```

**Review by PeterShaggyNoble — DISMISSED** [review]
As we have approval from the project maintainers, this one can be merged straight in.

**Review by adamrusted — APPROVED** [review]

**Pull request merged**
